config/locales/en.yml

```diff
@@ -1594,4 +1594,47 @@ en:
     zip: Zip
     zipcode: Zip Code
     zone: Zone
-    zones: Zones
\ No newline at end of file
+    zones: Zones
+
+---
+en:
+  spree:
+    navigator:
+      admin:
+        no_resource_found: No menu items found
+        sidebar: Navigation Manager
+        ajax:
+          new_menu: New Menu Item
+          menu_tree_error: The requested change has not been accepted and the tree has been returned to its previous state, please try again.
+          instructions: '* Right click an item to access the menu for adding, editing, deleting and renaming. Drag an item around to sort it.'
+        button:
+          back: Back to Menu Items
+          new: New Menu Item
+        table:
+          header:
+            name: Name
+            url: URL/Path
+        title:
+          index: Navigation Manager
+          new: New Menu Item
+          edit: 'Edit `%{name}` Menu Item'
+        fieldset:
+          legend:
+            new: New Menu Item
+            edit: Update Menu Item
+        label: &labels
+          name: Label
+          url: Path/URL
+          id: ID
+          class: Class
+          target: Target
+        flash:
+          success:
+            create: 'Menu item `%{name}` has been successfully created!'
+            update: 'Menu item `%{name}` has been successfully updated!'
+            destroy: 'Menu item `%{name}` has been successfully removed!'
+
+  activerecord:
+    models:
+      spree/navigator:
+        <<: *labels
```

Comment:
hey @amang321 I'm getting an error because of this line:

```
I18n::InvalidLocaleData: can not load translations from /dna/config/locales/en.yml: #<Psych::BadAlias: Unknown alias: labels>
```